packages/libretro/sameboy/package.mk

```diff
@@ -35,10 +35,11 @@ PKG_IS_ADDON="no"
 PKG_AUTORECONF="no"
 
 make_target() {
-  make -f Makefile.libretro
+  cd $PKG_BUILD/libretro
+  make -f Makefile
```

Comment:
Could likely use....
```
make_target() {
  make -C libretro
}
```

Save yourself having to change directory yourself.

Reply:
Will merge and fix it